### ISSUE TITLE
Bump JSON Schema version

### ISSUE THREAD

**Issue by pnrobinson** [phenopackets/phenopacket-tools#92]
https://github.com/networknt/json-schema-validator

V2020-12 -- our schema should be identical, but now networknt supports latest JSON Schema version

**Comment by pnrobinson** [comment]
@ielis Should we update?

**Comment by ielis** [comment]
The [JSON Schema 2020-12 Release Notes](https://json-schema.org/draft/2020-12/release-notes.html) highlights the changes between _Draft 2019-09_ and _Draft 2020-12_. 

There should be no issues with updating our schemas, because the changes do not affect our schemas. However, the authors of json-schema-validator, the library we use to validate phenopackets under the hood, are still working on correct implementation. 

In other words, if I update the `$schema`s to `[2020-12](https://json-schema.org/draft/2020-12/schema)` and use `SpecVersion.VersionFlag.V202012`, the tests fail. Specifically, all tests that validate an array member (e.g. one of phenotypic features).

I don't think we should update schema for the next release, let's stick with _Draft 2019-09_.
